.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+﻿name: CI Build
+on:
+    pull_request:
+        branches: [ $default-branch ]
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Add MSBuild to Path
+      uses: microsoft/setup-msbuild@v1.0.2
+
+    - name: Restore NuGet packages
+      working-directory: ${{env.GITHUB_WORKSPACE}}
+      run: nuget restore OpenCPPCoverage.sln
+
+    - name: Build solution (32 bit)
```

Comment:
It appears that the build needs to be done in this order, otherwise the github agent runs code analysis on the x64 solution endlessly.

Reply:
I am personally more worried about the prerequirement to the visx builder. Perhaps I'm wrong?

Reply:
My plan for this was to have it run as a build validation, no artifacts produced (I'm not sure how that works yet in github actions anyways). I'm more familiar with Azure DevOps pipelines, this seems a little harder to use so far.

I'm not sure if there's a way to validate the vsix (besides that part of the build passing).

Were you wanting to build the debug configuration as well?

Reply:
I know that github can create artefacts (we do it for fluidnc), but visx requires some installation etc. which makes it harder... Even though I applaud the effort, I'm personally not too concerned with it, Debug or not. The project has been very stable so far, with only a few releases and patches and seems to work across the board for c++ projects; in other words: it's not really an issue for me. Regardless, if it works I'd be happy to pull it.